spec/requests/sessions/login_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'POST api/v1/users/sign_in', type: :request do
+  let(:user) { create(:user) }
```

Comment:
Be explicit about the password
```suggestion
  let(:user) { create(:user, password: password) }
```

Reply:
what is the reason for this?

Reply:
If you rely on the factory's default values you run the risk of someone changing them and tests failing for what would appear to be a random reason. I'd say this is a case of tests not being dry, you have the same value repeated in two places so if I were to change one the other would break.

The other side of this argument is that if you decide to change password policies, let's say to require a symbol, you'd need to change many places. But I still prefer to be explicit especially for people reading the test in the future. For a small test like this, I can definitely assume that the password is the same as in the factory, but if it grows it'll be more dificult.